.github/workflows/backport.yml

```diff
@@ -34,6 +34,7 @@ jobs:
 
       - uses: actions/checkout@v4
         with:
+          persist-credentials: false
```

Comment:
`korthout/backport-action` may be using the persisted credentials:

https://github.com/korthout/backport-action/blob/main/src/git.ts

As far as I can tell, it isn't passing through its own `github_token` input to Git

Reply:
Should we simply revert the change in this file then:

```suggestion
```

Reply:
If I'm correct then that would be the easiest solution

Reply:
Let me test PR in my https://github.com/trueNAHO/stylix-playground fork.

Reply:
> Let me test this PR in my https://github.com/trueNAHO/stylix-playground fork.

I do not have the GitHub App setup and doing so might take too long. It might be safest to just remove revert the change in this file.